frontend/js/gather.js

```diff
@@ -1,18 +1,21 @@
-const name = document.querySelector('#name')
+const author = document.querySelector('#author')
 const poem = document.querySelector('#poem')
 const time = document.querySelector('#time')
 
-const url = 'package.json'
+const url = 'poems/poems.json'
 const req = { method: 'GET' }
 fetch(url, req)
   // turn json information into js object
   .then(res => res.text())
   // get data from poem database and print java script object "time"
   .then(data => {
     const obj = JSON.parse(data)
```

Comment:
here's how we can filter the array the way u described:
```js
    const obj = JSON.parse(data)
    // the variable below is the value for 10 minutes in milliseconds
    const min = 1000 * 60 * 10
    // this one is a timestamp for 10mins ago
    const past = Date.now() - min
    // here we use the array's "fitler" method to creaate a new array "sel"
    // which contains only the objcts with a datetime greater than 10mins ago
    const sel = obj.filter(p => p.datetime > past)  
 ```
keep in mind that the poems in the `poem.json` have `datetimes` all of which are more than 10mins ago (which means they'll all get filtered out && the `sel` array will be empty), but u can test this by changing their datetime values to something more recent (u can enter `Date.now()` in the browser's JavaScript console to create a timestamp for the current time)